src/juicy-popover.html

```diff
@@ -108,9 +100,14 @@
 
             var t = importDoc.querySelector('#juicy-popover-template');
             var clone = document.importNode(t.content, true);
-            clone.querySelector("#handle content").setAttribute("select", this.getAttribute("handleSelector") || ".handle");
-            clone.querySelector("#expandable content").setAttribute("select", this.getAttribute("expandableSelector") || ".expandable");
-            this.createShadowRoot().appendChild(clone);
+            var root;
+            if (this.attachShadow) {
+                root = this.attachShadow({ mode: 'open' });
+            } else {
+                root = this.createShadowRoot();
```

Comment:
I've added missing content elements. We also spoke about adding tests for this element, but I think it should be done as separated issue.